app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SessionDetailActivity.java

```diff
@@ -27,21 +37,31 @@ protected void onCreate(Bundle savedInstanceState) {
         getComponent().inject(this);
 
         final int sessionId = getIntent().getIntExtra(EXTRA_SESSION_ID, 0);
+        String parentClassName = getIntent().getStringExtra(EXTRA_PARENT);
+        try {
+            if (TextUtils.isEmpty(parentClassName)) {
+                parentClass = MainActivity.class;
+            } else {
+                parentClass = Class.forName(parentClassName);
+            }
+        } catch (ClassNotFoundException e) {
```

Comment:
I think this `try ~ catch` should be in `else` statement above, or it's okay to remove it.
What do you think?